R/zzz.R

```diff
@@ -7,6 +7,7 @@
 	.addPythonAndFoldersToSysPath(system.file(package="synapser"))
 	
 	.defineRPackageFunctions()
+	.defineOverloadFunctions()
```

Comment:
document that defineOverloadFunctions must come AFTER defineRPackageFunctions because it redefines selected generic functions.